spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'vanagon/platform'
 
+# These constants are defined for the purpose of the project/generic file merge tests
+# to point these directories to test areas under the /tmp directory.
+# This allows individual test cases to be specified accurately.
+# The actual resources/windows/wix files under vanagon are avoided, as the necessary
+# data structures are not available under the test conditions causing failures in the
+# ERB template translation
+
+WORK_BASE = "/tmp/vanwintest"
+VANAGON_ROOT = "#{WORK_BASE}/generic"
+PROJ_ROOT = "#{WORK_BASE}/project"
+WORKDIR = "#{WORK_BASE}/workdir"
+# Admittedly this might not be the best placed statement, but my limited rspec
+# started to defeat me when it came to using "let" for wixtestfiles
+WIXTESTFILES = File.join(FileUtils.getwd, "spec/fixtures/wix/resources/windows/wix")
```

Comment:
http://ruby-doc.org/core-2.1.0/File.html#method-c-expand_path will give you an absolute path.